components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderDAO.java

```diff
@@ -346,6 +355,19 @@ private Resource createResource(SAMLSSOServiceProviderDO serviceProviderDO) thro
                 resource.addProperty(IdentityRegistryResources.PROP_SAML_SLO_REQUEST_URL,
                         serviceProviderDO.getSloRequestURL());
             }
+            // Create doFrontChannelLogout property in the registry.
+            String doFrontChannelLogout = String.valueOf(serviceProviderDO.isDoFrontChannelLogout());
+            if (StringUtils.isEmpty(doFrontChannelLogout)){
+                resource.addProperty(IdentityRegistryResources.PROP_SAML_SSO_DO_FRONT_CHANNEL_LOGOUT, "false");
+            }
```

Comment:
Fix dormatting

Reply:
Fixed in master - https://github.com/wso2/carbon-identity-framework/blob/master/components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderDAO.java#L372